src/main/java/org/ehcache/sizeof/EhcacheSizeOfEngine.java

```diff
@@ -47,36 +44,26 @@ public Configuration getConfiguration() {
     /**
      * {@inheritDoc}
      */
-    public SizeOfEngine copyWith(int maxDepth, boolean abortWhenMaxDepthExceeded) {
-        return new EhcacheSizeOfEngine(this.cfg);
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    public Size sizeOf(final Object key, final Object value, final Object container) {
+    public Size sizeOf(final Object... objects) {
         Size size;
         try {
-            org.ehcache.sizeof.Size ourSize = sizeOf.deepSizeOf(cfg.getMaxDepth(), cfg.isAbort(), key, value, container);
+            org.ehcache.sizeof.Size ourSize = sizeOf.deepSizeOf(cfg.getMaxDepth(), cfg.isAbort(), objects);
             size = new Size(ourSize.getCalculated(), ourSize.isExact());
```

Comment:
I guess rewrapping and fqcn was there since this library depended on 2.8 as well which also had size.
So it was like getting sizeof Size and wrapping it to 2.8 size. I didn't realize once I removed the 2.8 dependency.